documentation/src/docs/user-guide.template.md

```diff
@@ -1295,6 +1295,55 @@ void restrictedEmailAddresses(@ForAll @Email(quotedLocalPart = false, ipv4Host =
 }
 ```
 
+### Date Arbitrary Types
+
```

Comment:
Add a sentence to describe that date generation is in an extra module which must be in a project's dependency.